extern/iso_c_fortran_bmi/test/test_iso_c.c

```diff
@@ -121,7 +121,7 @@ int main(int argc, char** argv)
     for(i = 0; i < count; i++){
         names[i] = malloc(sizeof(char)*BMI_MAX_VAR_NAME);
         //names[i] = "Hello World\0";
-        sprintf(names[i], "Hello World %d", i);
+        snprintf(names[i], sizeof(names[i]), "Hello World %d", i);
```

Comment:
This is wrong: `sizeof(names[i]) == sizeof(char*)`. The length should be `BMI_MAX_VAR_NAME` to match the allocation.

Reply:
It is a good catch, Phil. Thanks. Changes made.